src/contracts.rs

```diff
@@ -184,22 +184,23 @@ impl Contract {
     }
 
     /// Creates futures contract from specified symbol
-    pub fn futures(symbol: &str) -> Contract {
+    pub fn futures(local_symbol: &str, exchange: &str) -> Contract {
         Contract {
-            symbol: symbol.to_string(),
+            local_symbol: local_symbol.to_string(),
             security_type: SecurityType::Future,
             currency: "USD".to_string(),
+            exchange: exchange.to_string(),
             ..Default::default()
         }
     }
 
     /// Creates Crypto contract from specified symbol
-    pub fn crypto(symbol: &str) -> Contract {
+    pub fn crypto(symbol: &str, exchange: &str) -> Contract {
```

Comment:
I see a use case for this, but the `futures` and `crypto` methods have been around for a while. I prefer not to break the interface. Also with existing implementation allows you to specify the contract after using helper.
```rust
let mut contract = Contract::crypto("BTC")
contract.exchange = "PAXOS"
```
I'm okay with adding another method like `crypto_with_exchange`
```rust
let mut contract = Contract::crypto_with_exchange("BTC", "PAXOS")
```
What I have in mind for this but haven't gotten to implement is a contract builder. That would work something like this.
```rust
let contract = ContractBuilder::future()
                         .with_symbol("BTC")
                         .with_exchange("PAXOS")
                         .build()
```